### DISCUSSION TITLE
Typo in drop down menu

### DISCUSSION THREAD

**Discussion by tricantivu** [community/community#65260] [Mobile]
When adding a new shortcut, the label to select an organization is mispelled.

![image](https://github.com/community/community/assets/96702705/61109725-bd7a-4375-80b3-c4e1b206b16a)

*It should say organización*

As far as I know, this only happens if the application's language is Spanish.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Versión 1.127.2 (108489730)
Device: iPhone 11 (iOS 16.6)
</details>

**Comment by aconsuegra** [comment]
Gracias por el feedback @tricantivu!

I just reported the issue, and it should be fixed in one of our upcoming releases.

**Comment by tricantivu** [comment]
> Gracias por el feedback @tricantivu!
> 
> 
> 
> I just reported the issue, and it should be fixed in one of our upcoming releases. 

¡No hay problema! Y gracias por la respuesta rápida.
